tensorflow/python/keras/benchmarks/benchmark_util.py

```diff
@@ -108,19 +110,26 @@ def measure_performance(model_fn,
   avg_epoch_time_list, wall_time_list, exp_per_sec_list = [], [], []
   total_num_examples = epochs * num_examples
 
+  strategy = distribution_util.get_distribution_strategy(
+      distribution_strategy=distribution_strategy,
+      num_gpus=num_gpus)
+
   for _ in range(run_iters):
     timer = timeit.default_timer
-    t0 = timer()
-    model = model_fn()
-    build_time = timer() - t0
-
-    t1 = timer()
-    model.compile(
-        optimizer=optimizer,
-        loss=loss,
-        metrics=metrics,
-    )
-    compile_time = timer() - t1
+    # Init the distribution strategy scope for each iteration.
+    strategy_scope = distribution_util.get_strategy_scope(strategy)
+    with strategy_scope:
+      t0 = timer()
```

Comment:
If you put `t0` here, the computation of`wall_time` below is wrong. Let's make a new variable to record the build time, so it won't affect the wall_time.